src/main/webapp/components/FilterList.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div>
+    <!-- User can set their own filters. -->
+    <v-combobox v-model="filters"
+                label="Active Filters"
+                hint="Add your own"
+                multiple chips deletable-chips>
+    </v-combobox>
+    <!-- Or select from common filters -->
+    <v-checkbox v-model="filters"
+                v-for="filter in suggestedFilters" 
+                :key="filter"
+                :label="filter" 
+                :value="filter">
+    </v-checkbox>
+  </div>
+</template>
+
+<script>
+module.exports = {
+  props: {
+    value: Array,
+    school: String, // TODO: Modify SearchBar.vue to emit school and course to NoteGrid, and pass to FilterList
+    course: String,
+  },
+  data: function() {
+    return {
+      suggestedFilters: [
+        "MAC",
+        "ENC",
+        "COP",
+      ]
+    }
+  },
+  computed: {
+    filters: {
+      get: function() {
+        return this.value;
+      },
+      set: function(filters) {
+        this.$emit('input', filters)
+      }
+    },
+  },
+  methods: {
+    fetchSuggestedLabels: function() {
+      let url = new URL("/common-labels", window.location.href);
+      url.searchParams.set('school', this.school);
+      url.searchParams.set('course', this.course);
+
+      fetch(url)
+        .then(response => response.json())
+        .then(result => {
+          this.suggestedFilters = result;
+        })
```

Comment:
If this is a call to the backend to get all the school and course labels, the way I ended up designing the API was that you'd get back an object where the two keys are `"schools"` and `"courses"` and their values are String arrays of labels of those type. So you could either just combine those two arrays and make that the `suggestedFilters` or you could keep two variables `suggestedSchoolFilters` and `suggestedCourseFilters` for the two arrays; I'm not sure which of these options is more useful to you on the frontend. Feel free to add a review comment on #57 if you think that API should respond differently than it does.